cmd/create.go

```diff
@@ -228,7 +240,7 @@ var createCmd = &cobra.Command{
 		fmt.Println(endingMsgStyle.Render("\nNext steps:"))
 		fmt.Println(endingMsgStyle.Render(fmt.Sprintf("• cd into the newly created project with: `cd %s`\n", project.ProjectName)))
 
-		if options.Advanced.Choices["AddHTMXTempl"] {
+		if options.Advanced.Choices[string(flags.Htmx)] {
```

Comment:
I was running through this again. I like the idea of this change, but the choice is `"Htmx"` and `string(flags.Htmx)` returns `"htmx"` so these 'next steps' never get printed. 
```suggestion
		if options.Advanced.Choices["Htmx"] {
```